### PR TITLE
Feat/questionnaire politics community

### PR DESCRIPTION
**Pull request by felixmichel**
- [ ] Colors?
- [ ] Shares nochmal prüfen
- [ ] Other layout stuff and improvements welcome

### CODE REVIEW

File: apps/www/components/PoliticsCommunityQuestionnaire/Person.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+          if (!submission) {
+            if (process.browser) {
+              router.replace({ pathname: OVERVIEW_QUESTIONNAIRE_PATH })
+            }
+            return null
+          }
+
+          const meta = {
+            url,
+            title: '26 Fragen zur Politik',
```
Comment (jstcki) [inline apps/www/components/PoliticsCommunityQuestionnaire/Person.js:135]: Get this from config too?

File: apps/www/components/PoliticsCommunityQuestionnaire/Person.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                      textAlign: 'center',
+                    }}
+                  >
+                    <Figure
+                      size='tiny'
+                      attributes={{ style: { position: 'relative' } }}
+                    >
+                      <FigureImage src={QUESTIONNAIRE_SQUARE_IMG_URL} />
+                      <FigureCaption>
+                        <FigureByline>Cristina Spanò</FigureByline>
```
Comment (jstcki) [inline apps/www/components/PoliticsCommunityQuestionnaire/Person.js:164]: Are these credits correct? Also use config like for the image URL itself?

File: apps/www/components/PoliticsCommunityQuestionnaire/config.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+]
+
+export const EDIT_QUESTIONNAIRE_PATH = '/2023/07/12/politikfragebogen-community'
+export const OVERVIEW_QUESTIONNAIRE_PATH =
+  '/26-fragen-zur-politik-ihre-antworten'
+
+export const PERSON_PAGE_PATH = 'politikfragebogen-community'
+export const PERSON_SHARE_TEXT = '26 Fragen zur Politik – die Antworten von '
+
+export const QUESTIONNAIRE_SLUG = 'politiker-fragebogen-community'
```
Comment (jstcki) [inline apps/www/components/PoliticsCommunityQuestionnaire/config.ts:64]: A bit confusing that this is "politiker-fragebogen" and "politikfragebogen" elsewhere?

File: apps/www/components/Questionnaire/Submissions/QuestionFeatured.js [hunk 4]
```diff
@@ -237,7 +249,7 @@ export const QuestionFeatured = ({
       ref={questionRef}
       id={questions[0].id}
       style={{
-        padding: '0 0 46px 0',
+        padding: '30px 0 46px 0',
```
Comment (jstcki) [inline apps/www/components/Questionnaire/Submissions/QuestionFeatured.js:252]: Thanks for the fix 😄 But why not use the same padding as on the bottom?

File: apps/www/components/PoliticsCommunityQuestionnaire/Person.js [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+                  {...submissionStyles.header}
+                  style={{
+                    top: headerHeight,
+                    padding: '10px 0',
+                    margin: '10px 0 0',
+                  }}
+                  {...colorScheme.set('backgroundColor', 'default')}
+                >
+                  <div {...submissionStyles.headerText}>
+                    <NextLink href={OVERVIEW_QUESTIONNAIRE_PATH} passHref>
```
Comment (jstcki) [inline apps/www/components/PoliticsCommunityQuestionnaire/Person.js:196]: Probably we've done this elsewhere too but is IconButton a `<a>` element? If it's a `<button>` it works but is wrong HTML.

**Review by jstcki — APPROVED** [review]
A few details, otherwise LGTM!

File: apps/www/components/Questionnaire/Submissions/QuestionFeatured.js (hunk 4 above)
Comment (felixmichel) [reply apps/www/components/Questionnaire/Submissions/QuestionFeatured.js:252]: i don't know...

File: apps/www/components/PoliticsCommunityQuestionnaire/config.ts (hunk 3 above)
Comment (felixmichel) [reply apps/www/components/PoliticsCommunityQuestionnaire/config.ts:64]: yes, i fucked up the slug. But Sharon is going to kill me, if I tell her that i need another slug

File: apps/www/components/PoliticsCommunityQuestionnaire/Person.js (hunk 5 above)
Comment (felixmichel) [reply apps/www/components/PoliticsCommunityQuestionnaire/Person.js:196]: hmm. what shall i do then?

Comment (jstcki) [reply apps/www/components/PoliticsCommunityQuestionnaire/Person.js:197]: Nevermind, apparently [IconButton renders automagically as `<a>`](https://styleguide.republik.ch/components/iconbutton) when `href` is passed.

**Pull request merged**